OpenRA.Mods.Common/Traits/Render/WithStoresResourcesPipsDecoration.cs

```diff
@@ -42,26 +42,26 @@ public class WithHarvesterPipsDecorationInfo : WithDecorationBaseInfo, Requires<
 		[PaletteReference]
 		public readonly string Palette = "chrome";
 
-		public override object Create(ActorInitializer init) { return new WithHarvesterPipsDecoration(init.Self, this); }
+		public override object Create(ActorInitializer init) { return new WithStoresResourcesPipsDecoration(init.Self, this); }
 	}
 
-	public class WithHarvesterPipsDecoration : WithDecorationBase<WithHarvesterPipsDecorationInfo>
+	public class WithStoresResourcesPipsDecoration : WithDecorationBase<WithStoresResourcesPipsDecorationInfo>
 	{
-		readonly Harvester harvester;
+		readonly IStoresResources storesResources;
 		readonly Animation pips;
 
-		public WithHarvesterPipsDecoration(Actor self, WithHarvesterPipsDecorationInfo info)
+		public WithStoresResourcesPipsDecoration(Actor self, WithStoresResourcesPipsDecorationInfo info)
 			: base(self, info)
 		{
-			harvester = self.Trait<Harvester>();
+			storesResources = self.Trait<IStoresResources>();
```

Comment:
Plural is a bit confusing. `storeResourcesTrait` seems clearer. Harvester used to be clearer.

Reply:
I think the way it's currently named is the unspoken convention in this codebase